src/locales/de/battle.ts

```diff
@@ -14,6 +14,7 @@ export const battle: SimpleTranslationEntries = {
   "switchQuestion": "Möchtest du\n{{pokemonName}} auswechseln?",
   "trainerDefeated": "{{trainerName}}\nwurde besiegt!",
   "moneyWon": "Du gewinnst\n{{moneyAmount}} ₽!",
+  "moneyPickedUp": "You picked up ₽{{moneyAmount}}!",
```

Comment:
```suggestion
  "moneyPickedUp": "Du hebst {{moneyAmount}} ₽ auf!",
```

Source Pokémon Scarlet/Violet